metadata-db-app/src/cmr/metadata_db/migrations/088_add_meta_to_provider.clj

```diff
@@ -19,4 +19,4 @@
   []
   ;; do nothing as we don't want to loose the metadata documents when migrating
```

Comment:
nit: while you're here, can you fix the typo "loose"?

Reply:
fixed.